packages/zoe/test/unitTests/zcf/test-zoeHelpersWZcf.js

```diff
@@ -100,6 +103,102 @@ test(`zoeHelper with zcf - swap no match`, async t => {
   assertPayoutAmount(t, moolaIssuer, await bUserSeat.getPayout('A'), moola(5));
 });
 
+test(`zcf usesNatMath`, async t => {
+  const { zcf } = await setupZCFTest();
+  const zcfMint = await zcf.makeZCFMint('A');
+  const { brand } = zcfMint.getIssuerRecord();
+  t.falsy(assertUsesNatMath(zcf, brand), 'default');
```

Comment:
huh, why is this falsy? It seems like it should not throw, which should have a different assertion

Reply:
I really prefer using `notThrows` when our intention is to assert that the function call does not throw. Is there any reason not to? When I read `falsy`, I expect something that is logically not true. So although this technically works, it's not as clear as it could be from a readability perspective.

Reply:
done.